### PR TITLE
Update to Vert.x 4.5.8 and Netty 4.1.110

### PR DESCRIPTION
**Pull request by cescoffier**
- bump Mutiny Bindings to 3.13.0
- bump Quarkus HTTP to 5.3.0
- update Netty SSL released substitutions

Netty 4.1.111 causes a problem in gRPC. It seems to be a frame size issue. Need to be investigated.

### CODE REVIEW

File: extensions/netty/runtime/src/main/java/io/quarkus/netty/runtime/graal/NettySubstitutions.java [hunk 1]
```diff
@@ -9,11 +9,7 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.nio.ByteBuffer;
-import java.security.InvalidAlgorithmParameterException;
-import java.security.KeyException;
-import java.security.NoSuchAlgorithmException;
-import java.security.PrivateKey;
-import java.security.Provider;
+import java.security.*;
```
Comment (gsmet) [inline extensions/netty/runtime/src/main/java/io/quarkus/netty/runtime/graal/NettySubstitutions.java:12]: Nice try :).

**Review by gsmet — DISMISSED** [review]
Please check your IDE settings. A few star imports crawled in.

File: extensions/netty/runtime/src/main/java/io/quarkus/netty/runtime/graal/NettySubstitutions.java (hunk 1 above)
Comment (cescoffier) [reply extensions/netty/runtime/src/main/java/io/quarkus/netty/runtime/graal/NettySubstitutions.java]: pfff... Stupid IDE is losing the config every time I re-import the project.

**Review by geoand — APPROVED** [review]

**Comment by geoand** [comment]
cc @franz1981 who I know wants to try the new `AdaptivePoolingAllocator`

**Comment by cescoffier** [comment]
We would need to investigate the gRPC failure. My quick analysis indicates that it's a frame size issue. The response exceeded the default frame size.

**Comment by geoand** [comment]
Is that reflection test thing that was disabled testing a feature that is really obscure?

**Comment by cescoffier** [comment]
Yeah the reflection service from gRPC (which has nothing to do with  Java reflection) is a way to discover gRPc services.

However, it might fails because there is something else going on. We need to investigate.

**Comment by cescoffier** [comment]
@franz1981 @geoand @vietj So, the gRPC issue comes from an incompatibility between grpc java and NEtty 4.1.111. PRevious versions of Netty work :-(. Need to see which commit can causes such a problem.

We may have to stay on 4.1.110 for now.

**Review by geoand — APPROVED** [review]

**Comment by gastaldi** [comment]
The commit message seems misleading, given that Netty is no longer updated?

**Comment by gsmet** [comment]
@gastaldi Netty is updated but just to 4.1.110.

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
@cescoffier is it something we want to shoe in 3.12?

**Comment by gastaldi** [comment]
Ah doh, I missed the Netty bump, nvm

**Comment by cescoffier** [comment]
@gsmet no :-)

**Comment by franz1981** [comment]
FYI https://github.com/netty/netty/issues/12919#issuecomment-2168581233

**Comment by vietj** [comment]
interesting, thanks

**Comment by franz1981** [comment]
@cescoffier so final verdict https://github.com/grpc/grpc-java/issues/11284#issuecomment-2172355510

**Comment by vietj** [comment]
we need to check if there is anything TBD in vertx

**Pull request merged**